src/ngraph/graph_util.hpp

```diff
@@ -53,6 +53,11 @@ namespace ngraph
                         std::function<void(std::shared_ptr<Node>)> f,
                         bool include_control_deps);
 
+    void traverse_nodes(const NodeVector& subgraph_results,
```

Comment:
Would be nice to have a short comment on what this version of `traverse_nodes()` do that's different from others.

Reply:
Based on @adstraw feedback, i'm extending the current method instead of adding a new one. Will expand on the comment in the cpp file.